README.md

```diff
@@ -1,73 +1 @@
-* Protobuf Compiler
-    * Please install the protobuf compiler for your platform. On Ubuntu `sudo apt install protobuf-compiler`
-
-```
-git clone https://github.com/zingolabs/zingolib.git
-cd zingolib
-cargo build --release
-./target/release/zingo-cli --data-dir /path/to/data_directory/
-```
-
-This will launch the interactive prompt. Type `help` to get a list of commands.
-
-## Library:
-We currently depend on unreleased patches of orchard and librustzcash, via cargo's patching mechanism.
-Because patching only applies at the top level of a project, in order to use this library as a dependency, copy the `[patch.crates-io]` section of `Cargo.toml` into your own workspace `Cargo.toml`
-
-## Notes:
-* If you want to run your own server, please see [zingo lightwalletd](https://github.com/zingolabs/lightwalletd), and then run `./zingo-cli --server http://127.0.0.1:9067`
-* The default log file is in `~/.zcash/zingo-wallet.debug.log`. A default wallet is stored in `~/.zcash/zingo-wallet.dat`
-* Currently, the default, hard-coded `lightwalletd` server is https://mainnet.lightwalletd.com:9067/. To change this, you can modify the `DEFAULT_SERVER` const in `config/src/lib.rs`
-
-## Running in non-interactive mode:
-You can also run `zingo-cli` in non-interactive mode by passing the command you want to run as an argument. For example, `zingo-cli addresses` will list all wallet addresses and exit.
-Run `zingo-cli help` to see a list of all commands.
-
-## Options
-Here are some CLI arguments you can pass to `zingo-cli`. Please run `zingo-cli --help` for the full list.
-
-* `--data-dir`: uses the specified path as data directory. This is required when not using the `--regtest` option.
-    * Example: `./zingo-cli --data-dir /path/to/data_directory/` will use the provided directory to store `zingo-wallet.dat` and logs. If the provided directory does not exist, it will create it.
-* `--server`: Connect to a custom zcash lightwalletd server.
-    * Example: `./zingo-cli --data-dir /path/to/data_directory/ --server 127.0.0.1:9067`
-* `--seed`: Restore a wallet from a seed phrase. Note that this will fail if there is an existing wallet. Delete (or move) any existing wallet to restore from the 24-word seed phrase
-* `--birthday`: Specify wallet birthday when restoring from seed. This is the earlist block height where the wallet has a transaction.
-    * Example: `./zingo-cli --data-dir /path/to/data_directory/ --seed "twenty four words seed phrase" --birthday 1234567`
-* `--recover`: Attempt to recover the seed phrase from a corrupted wallet
-
-## Regtest
-There is an experimental feature flag available with `zingo-cli`, in which the cli works in regtest mode, by also locally running `zcashd` and `lightwalletd`.
-
-For a relatively recent user experience please see: https://free2z.cash/zingo-cli-in-regtest-mode
-
-Please see `zingolib/regtest/README.md` for a detailed explanation.
\ No newline at end of file
+see ./docs/README.md
```

Comment:
my preference is this readme file stays where it is so it appears on github front page properly.
having a doc directory for the other readmes makes sense though good thinking